nginx/hs_proxy/config/hydroshare-config.sh

```diff
@@ -0,0 +1,30 @@
+HS_PATH=/home/hydro/hydroshare
+HOST_SSL_DIR=/opt/hs-certs
+HS_DATABASE=pg.deploy.sql
+HS_LOG_FILES=/var/hydroshare/log
+HS_SERVICE_UID=20028
+HS_SERVICE_GID=10000
+USE_NGINX=true
+USE_SSL=true
+USE_SECURITY=false
+USE_HTTP_AUTH=false
+FQDN_OR_IP=www-hydroshare.edc.renci.org
+NGINX_DIR=/home/hydro/hydroshare/nginx
+SOLR_VM=db1-beta-hydroshare.edc.renci.org
```

Comment:
These are beta configs.  The idea behind checking this in is just to docuemtn the configurations, but it won't actually be used in deploys, correct?